opensource/doc-style.md

```diff
@@ -221,14 +221,14 @@ When writing CLI examples, give the user hints by making the examples resemble
 exactly what they see in their shell:
 
 * Indent shell examples by 4 spaces so they get rendered as code blocks.
-* Start typed commands with `$ ` (dollar space), so that they are easily
+* Start typed commands with <tt>$ </tt> (dollar space), so that they are easily
   differentiated from program output.
 * Program output has no prefix.
 * Comments begin with # (hash space).
-* In-container shell commands, begin with `$$ ` (dollar dollar space).
+* In-container shell commands, begin with <tt>$$ </tt> (dollar dollar space).
```

Comment:
Hi @rillig -- I see how it's rendering and it looks correct. The white space is there, rendered in monospace, with a grey background. It is using a `<code>` tag when rendered in HTML. Switching to a `<tt>` tag removes the grey background, but doesn't actually change the amount of whitespace, and in fact makes it less clear how much whitespace there is, because there is no visual delineation between the whitespace being specified in the text, and the white background of the page. Hope that makes sense. :)